c/meterpreter/source/metsrv/base.c

```diff
@@ -450,10 +459,9 @@ BOOL command_handle(Remote *remote, Packet *packet)
 			break;
 		}
 
-		baseCommand = command_locate_base(commandId);
```

Comment:
Seeing as the `command_locate_base` function is not being used anywhere else with this removal, it would be good to remove this function definition from the code, particularly given the changes that are being made to this code.